src/main/res/values/themes.xml

```diff
@@ -84,13 +87,13 @@
         <item name="android:textColorSecondary">@color/secondary_text</item>
     </style>
 
-    <style name="Theme.Amahi.ActionBar" parent="@style/Widget.AppCompat.ActionBar">
+    <style name="Theme.Amahi.ActionBar" parent="@style/ThemeOverlay.AppCompat.ActionBar">
```

Comment:
Can you explain what's happening here?

Reply:
I reverted it

Reply:
Actually, it was present in themes v21 so I changed `Widget.AppCompat.ActionBar` to `ThemeOverlay.AppCompat.ActionBar`. Nonetheless, `Widget` will also work fine.